.github/workflows/test.yml

```diff
@@ -1,8 +1,12 @@
-name: Run tests
+name: Build PR
 on:
-  push:
-    branches-ignore:
-      - master
+  pull_request_target:
```

Comment:
er denne safe? Burde være `pull_request`?

Reply:
enig! husker ikke detaljene men uten `pull_request_target` så kan det ble litt merkelig oppførsel ved bygging. Mulig GHA sjekker ut master branchen isteden for inneværende branch. Jeg tror det er greit å beholde det slik det var, on push, hvis det er bare nav folk som kan lage direkte PR